macros/SpecName.ejs

```diff
@@ -142,7 +142,7 @@ var specList = {
         url  : "https://drafts.csswg.org/css-box-3/"
     },
     'CSS3 Box Alignment':{
-        name : 'CSS Box Alignment Module',
+        name : 'CSS Box Alignment Module Level 3',
         url  : "https://drafts.csswg.org/css-align/"
```

Comment:
Can you please adjust the URL, too? It should rather be https://drafts.csswg.org/css-align-3/.

Sebastian